contracts/payloads/IGP15/PayloadIGP15.sol

```diff
@@ -0,0 +1,668 @@
+    function action3() internal {
+        deploy_sUSDe_USDC_VAULT();
+    }
+
+    /// @notice Action 4: Deploy sUSDe/USDT vault and set related configs.
+    function action4() internal {
+        deploy_sUSDe_USDT_VAULT();
+    }
+
+    /// @notice Action 5: Update market rates for USDC.
+    function action5() internal {
+        AdminModuleStructs.RateDataV2Params[]
+            memory params_ = new AdminModuleStructs.RateDataV2Params[](1);
+
+        params_[0] = AdminModuleStructs.RateDataV2Params({
+            token: USDC_ADDRESS, // USDC
+            kink1: 80 * 1e2, // 80%
+            kink2: 93 * 1e2, // 93%
+            rateAtUtilizationZero: 0, // 0%
+            rateAtUtilizationKink1: 10 * 1e2, // 10%
+            rateAtUtilizationKink2: 15 * 1e2, // 15%
+            rateAtUtilizationMax: 25 * 1e2 // 25%
+        });
+
+        LIQUIDITY.updateRateDataV2s(params_);
+    }
+
+    /// @notice Action 6: Update market rates for USDT.
+    function action6() internal {
+        AdminModuleStructs.RateDataV2Params[]
+            memory params_ = new AdminModuleStructs.RateDataV2Params[](1);
+
+        params_[0] = AdminModuleStructs.RateDataV2Params({
+            token: USDT_ADDRESS, // USDT
+            kink1: 80 * 1e2, // 80%
+            kink2: 93 * 1e2, // 93%
+            rateAtUtilizationZero: 0, // 0%
+            rateAtUtilizationKink1: 10 * 1e2, // 10%
+            rateAtUtilizationKink2: 15 * 1e2, // 15%
+            rateAtUtilizationMax: 25 * 1e2 // 25%
+        });
+
+        LIQUIDITY.updateRateDataV2s(params_);
+    }
+
+    /// @notice Action 7: Update rewards for fUSDC.
+    function action7() internal {
+        IFTokenAdmin(F_USDC).updateRewards(
+            0x6CC89782495A2162b2A4f5b206E2A06Dc8675090
+        );
+    }
+
+    /// @notice Action 8: Update rewards for fUSDT.
+    function action8() internal {
+        IFTokenAdmin(F_USDT).updateRewards(
+            0x6CC89782495A2162b2A4f5b206E2A06Dc8675090
+        );
+    }
+
+    /***********************************|
+    |     Vault Deployment Helper       |
+    |__________________________________*/
+    function deploy_sUSDe_USDC_VAULT() internal {
+        // Deploy sUSDe/USDC vault.
+        address vault_ = VAULT_T1_FACTORY.deployVault(
+            address(VAULT_T1_DEPLOYMENT_LOGIC),
+            abi.encodeWithSelector(
+                IFluidVaultT1DeploymentLogic.vaultT1.selector,
+                sUSDe_ADDRESS, // sUSDe,
+                USDC_ADDRESS // USDC
+            )
+        );
+
+        // Set user supply config for the vault on Liquidity Layer.
+        {
+            AdminModuleStructs.UserSupplyConfig[]
+                memory configs_ = new AdminModuleStructs.UserSupplyConfig[](1);
+
+            configs_[0] = AdminModuleStructs.UserSupplyConfig({
+                user: address(vault_),
+                token: sUSDe_ADDRESS,
+                mode: 1,
+                expandPercent: 25 * 1e2, // 25%
```

Comment:
in forum post this is 20%. although that might be wrong if we keep our standard as used for other vaults of 25%. https://gov.instadapp.io/t/launch-susde-usdc-and-susde-usdt/713

Reply:
same for USDT

Reply:
It's effectively 20%. With withdrawal gap!